main.go

```diff
@@ -46,14 +47,24 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
+	var (
+		metricsAddr          string
+		enableLeaderElection bool
+		useNodeAnnotation    bool
+	)
+
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.BoolVar(&useNodeAnnotation, "usenodeannotation", false,
```

Comment:
`usenodeannotation` must be passed as a CR field not as operator parameter.

Reply:
@mishoyama changed